eng/versioning/external_dependencies.txt

```diff
@@ -225,6 +225,7 @@ groovy3_org.spockframework:spock-core;2.0-groovy-3.0
 org.testng:testng;7.3.0
 uk.org.lidalia:slf4j-test;1.2.0
 com.google.truth:truth;1.1.3
+io.grpc:grpc-api;1.44.0
```

Comment:
the otel exporters no longer have a dependency on grpc library, hoping we can undo this change and below?

Reply:
The examples in the repo uses ManagedChannel from grpc library. Had to add this as test dependency

Reply:
let's change the examples, otel-java is not recommending to bring your own ManagedChannel anymore, as it will use OkHttp for transport now if you do not